src/actions/admin.js

```diff
@@ -6,7 +6,9 @@ import {General} from 'constants';
 
 import {Client4} from 'client';
 
-import {bindClientFunc} from './helpers';
+import {getLogErrorAction} from './errors';
```

Comment:
I know, I haven't even looked at this PR in days. I'll mention you when it's ready for re-review